cmd/tea/oapi_compose_test.go

```diff
@@ -56,7 +54,26 @@ func Test_runOAPICompose(t *testing.T) {
 				t.Errorf("runOAPICompose() error = %v, wantErr %v", err, tt.wantErr)
 				return
 			}
+			defer func() {
+				if cleanupAfterTest {
+					require.NoError(t, os.Remove(outputFilePath))
+				}
+			}()
 			assert.NoError(t, tt.cond(t))
+
+			tofp, err := os.ReadFile(tt.args.testOutputFilePath)
+			if err != nil {
+				t.Errorf("runOAPICompose() error = %v, wantErr %v", err, tt.wantErr)
```

Comment:
expected output file reading should never fail (misleading error message, its not a result of `runOAPICompose` at all)